common/rakefiles/entrypoint-common.rake

```diff
@@ -0,0 +1,85 @@
+desc "[ONLY ADMIN] Initialize GCP provider where all the projects will live"
+task :infra_init => [:set_vars, @gcp_creds_file] do
+  # Steps to initialize GCP with a minimum set of resources to allow Terraform
+  # create the rest of the infrastructure.
+  # These steps are the same found in this tutorial:
+  # https://cloud.google.com/community/tutorials/managing-gcp-projects-with-terraform
+  #
+  # Due to the needed permissions for creating the following resources, only an
+  # administrator of the organization can run this task.
+
+  if @project_type != "common" or @env != "prd"
+    puts "infra_init must run inside common/live/prd"
+    exit
+  end
+
+  output = `#{@exekube_cmd} gcloud projects list --format='json' --filter='name:#{ENV["TF_VAR_project_id"]}'`
+  hash = JSON.parse(output)
+  if hash.empty?
+    puts "#{ENV["TF_VAR_project_id"]} not found, I'll create a new one"
+    sh "
+      #{@exekube_cmd} gcloud projects create #{ENV["TF_VAR_project_id"]} \
```

Comment:
This project must be persistent, so I think that we don't need to create a random id for it. But we can discuss it if you don't believe so.

Reply:
We don't have to, but if we don't we can't test the whole thing - I would vote for consistent behavior with individual projects. But happy to discuss further.